azurelinuxagent/ga/exthandlers.py

```diff
@@ -178,6 +180,7 @@ def __init__(self):
         self.log_report = False
         self.log_etag = True
         self.log_process = False
+        self._global_force_enable = True # Extensions are always enabled on first start of the agent.
```

Comment:
I would probably flip this flag so the default value matches the default state, e.g. extension_enabled